azext_iot/central/_help.py

```diff
@@ -542,6 +582,36 @@ def _load_central_users_help():
             --object-id {objectId}
             --role operator
     """
+
+    helps[
+        "iot central user update"
+    ] = """
+        type: command
+        short-summary: Update roles for a user in the application.
+        long-summary: |
+                    Update a user with a different role.
+                    Updating the tenantId or objectId for a service principal user is not allowed.
+                    Updating the email address for an email user is not allowed.
+
+        examples:
+        - name: Update a user by email in the application
+          text: >
+            az iot central user update
+            --user-id {userId}
+            --app-id {appId}
+            --email {emailAddress}
```

Comment:
To add on, can we generate an email address using the user-id if we still want the email address?

Reply:
good point. I just keep user-id as the only parameter to pick the user. since "roles" is the only allowed field to be updated, both "email" and "servicePrincipal" users will have the same update syntax